make/release.mk

```diff
@@ -35,6 +41,17 @@ release-artifacts-signed: release-artifacts release-manifests-signed
 release: release-artifacts-signed
 	$(MAKE) --no-print-directory bin/release/metadata.json
 
+.PHONY: upload-release
+## Create a complete release and then upload it to a target GCS bucket specified by
+## RELEASE_TARGET_BUCKET
+##
+## @category Release
+upload-release: release | bin/tools/rclone
+ifeq ($(strip $(RELEASE_TARGET_BUCKET)),)
+	$(error Trying to upload-release but RELEASE_TARGET_BUCKET is empty)
+endif
+	./bin/tools/rclone copyto ./bin/release :gcs:ad-cm-test-release/stage/gcb/release/$(RELEASE_VERSION)
```

Comment:
> ad-cm-test-release

Is this a leftover from testing the new target?

Reply:
hahahaha I guess this is a great example of why we have code reviews, I was _totally_ blind to this!

Will fix, thanks :+1: